Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,64 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+            /* Splitting the equation into three parts A,B,C */
+
+            string[] token = equation.Split("=");
+            string[] equ = token[0].Split("*");

```

Comment:
Null check on token and token[0].
Change 'equ' to some better name, variable name should clearly say what data it's gonna hold.